homeassistant/brands/prusa.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
There is no need to add this, as there is only a single integration

Reply:
Thanks for the remark. I was trying it locally and while resolving the wrong commit email it must have slipped by.

Prusa as a brand is removed.